Trace context is required for transactions (#11456)
getsentry/sentry-docs: develop-docs/sdk/event-payloads/contexts.mdx

## Trace Context

> Additional information that allows Sentry to connect multiple transactions, spans, and/or errors into one trace. Trace context is required for transactions which will be dropped by relay if missing.

Author: ```suggestion
Additional information that allows Sentry to connect multiple transactions, spans, and/or errors into one trace. Trace context is required for transactions which will be dropped by relay if trace context is missing.
```

> Additional information that allows Sentry to connect multiple transactions, spans, and/or errors into one trace. Trace context is required for transactions which will be dropped by relay if trace context is missing.

Reviewer: ```suggestion
Additional information that allows Sentry to connect multiple transactions, spans, and/or errors into one trace. **Important:** If the trace context is missing, relay will drop the transaction. 
```
